packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -867,6 +879,24 @@ if (missingLangs.length) {
 const areDistResources = conf.get('static_directory') === 'dist';
 conf.set('are_dist_resources', areDistResources);
 
+if (conf.get('ecosystem_anon_id.keys_file')) {
+  const keysFile = path.resolve(
+    __dirname,
+    '..',
+    'config',
+    conf.get('ecosystem_anon_id.keys_file')
+  );
+  // Alter `ecosystem_anon_id.key` if the keys file exists.
+  if (fs.existsSync(keysFile)) {
+    const keys = JSON.parse(fs.readFileSync(keysFile));
```

Comment:
Might want to try/catch this line, since reading or parsing might fail.

Reply:
If I add that, our linter complains that it's an unnecessary try/catch. I just tried adding a file here with some jargon in an array and get an "unexpected token" error thrown at me from `JSON.parse` with a callstack so I think we're covered here. 👍